packages/components/addon/components/hds/side-nav/index.hbs

```diff
@@ -0,0 +1,38 @@
+<div class={{this.classNames}} ...attributes {{focus-trap isActive=this.shouldTrapFocus}}>
+  {{#if this.hasA11yRefocus}}
+    <NavigationNarrator
+      @routeChangeValidator={{@a11yRefocusRouteChangeValidator}}
+      @skipTo="#{{@a11yRefocusSkipTo}}"
+      @skipText={{@a11yRefocusSkipText}}
+      @navigationText={{@a11yRefocusNavigationText}}
+    />
+  {{/if}}
+
+  {{#if this.isResponsive}}
+    {{! template-lint-disable no-invalid-interactive}}
+    <div class="hds-side-nav__overlay" {{on "click" this.toggleMinimizedStatus}} />
```

Comment:
Just curious why this needs to be a `div` vs. a `button`?

Reply:
no strong reasons, apart from:
- I've ported as is from the existing Cloud UI implementation
- it would be weird to have a huge button covering the entire page (I know it's silly, but the idea makes me nervous)